scripts/workflow/test-docker.sh

```diff
@@ -18,9 +25,10 @@ if [[ $? -ne 0 ]]; then
   exit 1
 fi
 
-curl https://cdn.jsdelivr.net/gh/mogland/core@main/docker-compose.yml > docker-compose.yml
+curl https://raw.githubusercontent.com/mogland/core/main/docker-compose.yml > docker-compose.yml
+
+curl https://raw.githubusercontent.com/mogland/core/main/env.yaml > env.yaml
```

Comment:
这个地方其实不用这么搞，直接 touch 一个就好了
```suggestion
touch env.yaml
```